src/main/java/duke/command/TickCommand.java

```diff
@@ -0,0 +1,23 @@
+package duke.command;
+
+import duke.TaskList;
+import duke.Ui;
+import duke.Storage;
+import duke.task.Task;
+public class TickCommand extends Command{
+    private int taskIndex;
+
+    public TickCommand(int taskIndex) {
+        this.taskIndex = taskIndex;
+    }
+    @Override
```

Comment:
```suggestion
    }
    
    @Override
```
Whitespace here too! :)